05 - coder-files/src/App.jsx

```diff
@@ -1,16 +1,15 @@
+import { useState } from "react";
+import Button from "./components/Button";
+
 export default function App() {
+  const [counts, seCounts] = useState(0);
```

Comment:
Hey @riyoneri, this is an optional comment. but I think it is good practice to name variables in singular as long as it is not a collection of counts. like `counts` could `count`. Do you feel me?